### PR TITLE
[Android/Build] option to get sdk path @open sesame 03/27 16:25

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. add option to get gst/nns root dir and android sdk path.
2. check instrument test result if test is enabled.
3. prepare ci build.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: api/android/build-android-lib.sh [hunk 1]
```diff
@@ -148,15 +157,26 @@ check_package sed
...
 # Android SDK (Set your own path)
-[ -z "$ANDROID_HOME" ] && echo "Need to set ANDROID_HOME." && exit 1
+if [[ -z $android_sdk_dir ]]; then
+    [ -z "$ANDROID_HOME" ] && echo "Need to set ANDROID_HOME." && exit 1
+    android_sdk_dir=$ANDROID_HOME
+fi
 
-echo "Android SDK: $ANDROID_HOME"
+if [[ -z $android_ndk_dir ]]; then
+    android_ndk_dir=$android_sdk_dir/ndk-bundle
```
Comment (helloahn) [inline api/android/build-android-lib.sh]: is it a concrete path `ndk-bundle`? In my case, ndk has not been installed there.

Comment (jaeyun-jung) [reply api/android/build-android-lib.sh]: https://developer.android.com/studio/command-line/variables

I will change to ANDROID_SDK_ROOT / ANDROID_NDK_ROOT

**Comment by jaeyun-jung** [comment]
After merging this PR, I will add new PR (TAOS-CI) to build libraries using this script.

```
# Build Android library
bash ./api/android/build-android-lib.sh $common_option --build_type=all >> $api_build_log
result=$(($result+$?))

bash ./api/android/build-android-lib.sh $common_option --build_type=single >> $api_build_log
result=$(($result+$?))
```

build result is [here](http://nnsuite.mooo.com/nnstreamer/ci/repo-workers/pr-checker/2213-202004011215210.057553052902222-28bdc1f38c36d649c71ee8d72e186f8461d25c59/binary_repository/ANDROID/)

File: api/android/build-android-lib.sh [hunk 2]
```diff
@@ -30,28 +31,28 @@
...
-enable_snap='yes'
+enable_snap="yes"
 
 # Enable tensorflow-lite
-enable_tflite='yes'
+enable_tflite="yes"
 
 # Set tensorflow-lite version (available: 1.9 and 1.13)
-nnstreamer_tf_lite_ver='1.13'
+nnstreamer_tf_lite_ver="1.13"
```
Comment (myungjoo) [inline api/android/build-android-lib.sh:55]: Be prepared to add 1.15.2 soon (for next version, mid-April).
The next reference TF-Lite is 1.15.2 (using the forked one from tizen.org)

**Comment by myungjoo** [comment]
The next step is... to provide various build scripts or build-script-calling-script for the various configurations??

**Comment by jaeyun-jung** [comment]
After merging this PR,
1. change TAOS-CI : build using this script.
```
# Build Android library
bash ./api/android/build-android-lib.sh $common_option --build_type=all >> $api_build_log
result=$(($result+$?))

bash ./api/android/build-android-lib.sh $common_option --build_type=single >> $api_build_log
result=$(($result+$?))
```

2. change tf-lite version
Now I am building tf-lite 1.15.2 for android.

**Comment by wooksong** [comment]
> 2. change tf-lite version
>    Now I am building tf-lite 1.15.2 for android.

I am looking forward to it.

**Comment by jaeyun-jung** [comment]
https://github.com/nnstreamer/nnstreamer-android-resource/pull/2
tf-lite 1.15.2 uploaded.

Now testing new version in my local, and update this PR for new version.

File: api/android/build-android-lib.sh (hunk 2 above)
Comment (jaeyun-jung) [reply api/android/build-android-lib.sh]: Now set tf-lite version 1.15.2

**Pull request merged**